### PR TITLE
makes snakes extracting a new reagent snake venom

### PR DESCRIPTION
**Pull request by boodaliboo**
gives snakes a cool reason to be bought
use a bucket to extract venom from a snake
and adds a new snake venom reagent that does 2 toxin damage
:cl:  
rscadd: Added milking snakes for toxin and snake venom
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
We’re not milking snakes

**Comment by ToasterBiome** [comment]
maybe we can have a little snake milking?

**Comment by boodaliboo** [comment]
> maybe we can have a little snake milking?

i can milk it a different chemical

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Reflavor to extracting

**Comment by boodaliboo** [comment]
> Reflavor to extracting



> Reflavor to extracting
but you can milk snakes

**Comment by Bibby0110** [comment]
> > Reflavor to extracting
> 
> > Reflavor to extracting
> > but you can milk snakes

The professional terminology is “extracting venom” so that’s the one we’ll be using

**Comment by JamieD1** [comment]
Do not call it milking. Jesus christ man.

**Comment by Marmio64** [comment]
ive heard it called milking before a lot

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -23,6 +23,13 @@
 	toxpwr = 2.5
 	taste_description = "mushroom"
 
+/datum/reagent/toxin/snakevenom
+	name = "Snake Venom"
+	description = "A potent poison gotten from milking a snakes venom."
```
Comment (VaelophisNyx) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: "A potent venom extracted from a live snake."

Comment (boodaliboo) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: fine

**Comment by boodaliboo** [comment]
its extracting now

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Make custom text output for the udder, since we're extracting and not milking.

**Comment by Bibby0110** [comment]
Stale + conflicts. Reopen when ready to merge

**Pull request closed without merging**